public/locales/es/common.json

```
@@ -128,6 +128,8 @@
   "Volunteers": "Voluntarios",
   "unblock": "Desbloquear",
   "unblockedSuccessfully": "{{item}} desbloqueado exitosamente",
+  "pluginSettings": "Configuración de complementos",
```

Biome (2.1.2)

[error] The key pluginSettings was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)